File: Lagerstyringsprojekt/LagerSystem/LagersystemClassLibrary/Models/DeviceOverview.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LagerstyringClassLibrary.Models
{
    public class DeviceOverview
    {
        public DeviceOverview() { }

        [Key]
        public int id { get; set; }  // seq

        [Required]
        public string model { get; set; }  // changed to model
        public int device_type { get; set; }  // FK DeviceType
        public string image { get; set; }
        public int qty { get; set; }
        public int available_qty { get; set; }
        public DateTime last_ordered { get; set; }

        // Navigation property
        public DeviceType DeviceType { get; set; }

        // 1-m nav prop
        public ICollection<SingleDevice> Devices { get; set; } = new List<SingleDevice>();
    }
}

File: Lagerstyringsprojekt/LagerSystem/MainFrontEnd/src/Interfaces/DeviceOverview.ts
export interface IAllDeviceOverview {
    id: number;
    model: string;
    device_type: number;
    qty: number;
    available_qty: number;
    last_ordered: string;
}

export interface IUpdateDeviceOverview {
    id: number;
    model: string;
    device_type: number;
    image: string;
    qty: number;
    last_ordered: string;
}